### PR TITLE
test

### PR DESCRIPTION
**Pull request by amyleblond**
Changes: 

**Readme** : reordered our e2e testing
**favicon** : cleaned up the edges of the icon
**style.css** : made font consistent (Roboto) across the system
**course_list.html** : added a table with steps to make it more clear
**upload_front_matter.html** : added a table to make more clear
**logout.html** : made to look like login page (admin css)
**instructor_preference.html** : added a 'back' button for the AC view (back to ranking status page)
**ranking status.html**: changed instructor preference links to not open in new tab - makes back button addition to instructor_preference.html more useful
**home.html** : went in more detail with descriptions
**other .html files** : adding class to buttons, minor changes to formatting

### CODE REVIEW

File: taworks/taform/templates/taform/base.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <link rel="shortcut icon" href="{{ STATIC_URL }}/static/favicon/favicon.ico">
 <header id = "banner"></header>
 <head>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
```
Comment (j2kan) [inline taworks/taform/templates/taform/base.html]: can we do @import url(fonts.css);
in our style.css similar to how admin currently does it

check
static/admin/css/base.css
and
static/admin/css/fonts.css

File: taworks/taworks/settings.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = 'ri_*-e$mdbw&q#h2)epdhxi7(@026fir_f(a77)e)%p9t1+lxs'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (j2kan) [inline taworks/taworks/settings.py]: nada.

**Review by j2kan — COMMENTED** [review]
great work! just a few comments otherwise LGTM

**Review by smwatts — COMMENTED** [review]
Looks AMAZING @amyleblond, great job. Only comment is this button on the "Intro" page:
screen shot 2018-02-24 at 3 41 49 pm

And low priority, but if possible could we add padding on this page:
screen shot 2018-02-24 at 3 46 54 pm

**Review by j2kan — APPROVED** [review]
LGTM

**Comment by j2kan** [comment]
feel free to merge this and do the logout page padding on a separate PR as well.

**Pull request merged**